flytekit/common/workflow.py

```diff
@@ -8,19 +8,21 @@
 from flytekit.common import interface as _interface, nodes as _nodes, sdk_bases as _sdk_bases, \
     launch_plan as _launch_plan, promise as _promise
 from flytekit.common.core import identifier as _identifier
+from flytekit.common.tasks import task as _task
 from flytekit.common.exceptions import scopes as _exception_scopes, user as _user_exceptions
 from flytekit.common.mixins import registerable as _registerable, hash as _hash_mixin
 from flytekit.common.types import helpers as _type_helpers
+from flytekit.common.promise import Input as _Input
 from flytekit.configuration import internal as _internal_config
 from flytekit.engines import loader as _engine_loader
-from flytekit.models import interface as _interface_models, literals as _literal_models,\
+from flytekit.models import interface as _interface_models, literals as _literal_models, \
     launch_plan as _launch_plan_models
 from flytekit.models.core import workflow as _workflow_models, identifier as _identifier_model
 
 
 class Output(object):
 
-    def __init__(self, name, value, sdk_type=None, help=None):
+    def __init__(self, name, value, sdk_type=None, binding_data=None, help=None):
```

Comment:
I think we can avoid changing the definition of this constructor to have divergent. See comment below. If not, we should place`_binding_data`  after `help` and precede with an underscore since it's a private constructor behavior.